### PR TITLE
Created new pull request template

### PR DESCRIPTION
**Pull request by lanseafood**
Should pre-load the description of a PR when created, users can fill out or delete appropriate sections. Open to adjustments on what a proper PR template should look like

### CODE REVIEW

File: pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+| ----------- | ----------- |
+| EPIC ⚡| [Oceanwater-XXX](url) |
+| Github :octocat:  | # |
+
+
+## Summary of Changes
+* 
+
+## Test
+*
```
Comment (kmdalal) [inline pull_request_template.md:12]: Dumb question: where is this file?

**Comment by kmdalal** [comment]
In general 'hotfix' branches are intended for merge into master and not melodic-devel.   No big deal though.

File: pull_request_template.md (hunk 1 above)
Comment (lanseafood) [reply pull_request_template.md:12]: i believe it lives top level of ow_simulator! @kmdalal

Comment (kmdalal) [reply pull_request_template.md:12]: I don't see this file, or the branch.  I'm approving based on the diff - looks good.

**Review by kmdalal — APPROVED** [review]

File: pull_request_template.md (hunk 1 above)
Comment (lanseafood) [reply pull_request_template.md:12]: @kmdalal You can see it if you go to the repo and switch the view from master to this hotfix branch. See screen shot (at the bottom). It's a completely new file so you wouldn't see it on master or anywhere else :) Will be merging!

image

**Comment by mogumbo** [comment]
This is merged, but its only 16 hours old and this is the first chance I have had to look at it.

**Comment by lanseafood** [comment]
It's actually 4 days old, it was just merged about 17 hrs ago.

Lanssie Mingyue Ma
U.C. Berkeley B.A. Computer Science
Georgia Tech M.S. CSE
Georgia Tech Ph.D. CSE/AE

**Comment by Samahu** [comment]
I think having the `pull_request_template.md` at the main directory is not the right place. I recommend moving this into a subfolder maybe under `oceanwaters` folder.

**Pull request merged**